From ef2e9b81c6515e138f4feed08d9e3d8965e4a453 Mon Sep 17 00:00:00 2001
From: Dominik Heidler <dheidler@suse.de>
Date: Mon, 16 Jun 2025 14:50:11 +0200
Subject: [PATCH] Add prusaslicer url scheme handling to .desktop file

On printables.com the "open in prusaslicer" button requires this url schema to be registered.
---
 src/platform/unix/PrusaSlicer.desktop | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/platform/unix/PrusaSlicer.desktop b/src/platform/unix/PrusaSlicer.desktop
index ead8539eeaf..5dcf40ea015 100644
--- a/src/platform/unix/PrusaSlicer.desktop
+++ b/src/platform/unix/PrusaSlicer.desktop
@@ -5,7 +5,7 @@ Icon=PrusaSlicer
 Exec=prusa-slicer %F
 Terminal=false
 Type=Application
-MimeType=model/stl;application/vnd.ms-3mfdocument;application/prs.wavefront-obj;application/x-amf;
+MimeType=model/stl;application/vnd.ms-3mfdocument;application/prs.wavefront-obj;application/x-amf;x-scheme-handler/prusaslicer;
 Categories=Graphics;3DGraphics;Engineering;
 Keywords=3D;Printing;Slicer;slice;3D;printer;convert;gcode;stl;obj;amf;SLA
 StartupNotify=false
